tests/test_scene_optimizer.py

```diff
@@ -87,60 +62,52 @@ def test_find_largest_connected_component(self):
         (
             computed_relative_rotations,
             computed_relative_unit_translations,
-        ) = select_largest_connected_component(
-            input_relative_rotations, input_relative_unit_translations
-        )
+        ) = select_largest_connected_component(input_relative_rotations, input_relative_unit_translations)
 
         # check the edges in the pruned graph
-        self.assertCountEqual(
-            list(computed_relative_rotations.keys()), expected_edges
-        )
-        self.assertCountEqual(
-            list(computed_relative_unit_translations.keys()), expected_edges
-        )
+        self.assertCountEqual(list(computed_relative_rotations.keys()), expected_edges)
+        self.assertCountEqual(list(computed_relative_unit_translations.keys()), expected_edges)
 
         # check the actual Rot3 and Unit3 values
         for (i1, i2) in expected_edges:
+            self.assertTrue(computed_relative_rotations[(i1, i2)].equals(input_relative_rotations[(i1, i2)], 1e-2))
             self.assertTrue(
-                computed_relative_rotations[(i1, i2)].equals(
-                    input_relative_rotations[(i1, i2)], 1e-2
-                )
-            )
-            self.assertTrue(
-                computed_relative_unit_translations[(i1, i2)].equals(
-                    input_relative_unit_translations[(i1, i2)], 1e-2
-                )
+                computed_relative_unit_translations[(i1, i2)].equals(input_relative_unit_translations[(i1, i2)], 1e-2)
             )
 
     def test_create_computation_graph(self):
         """Will test Dask multi-processing capabilities and ability to serialize all objects."""
         use_intrinsics_in_verification = False
 
-        # generate the dask computation graph
-        sfm_result_graph = self.obj.create_computation_graph(
-            len(self.loader),
-            self.loader.get_valid_pairs(),
-            self.loader.create_computation_graph_for_images(),
-            self.loader.create_computation_graph_for_intrinsics(),
-            use_intrinsics_in_verification=use_intrinsics_in_verification,
-        )
+        with initialize_config_module(config_module="gtsfm.configs"):
```

Comment:
why are we doing this in a test function and not the setup?

Reply:
using a scope like this was the idiomatic way proposed in the [Hydra unit test examples](https://github.com/facebookresearch/hydra/tree/master/examples/advanced/hydra_app_example/tests#L29), so I followed it. But we couldn't have the scope if we set it up in `setUp(self)`.